README.md

```diff
@@ -1 +1,85 @@
-# CapstoneJS
\ No newline at end of file
+# CapstoneJS
+
+In this project we will prepare a Kanban board with a GitHub project, with my Learning partner. It's javascript capstone project with external API..
+
+![image] ![screenshot](photo.jpg)
+
+## Video link
+
+[loom Link] (https://loom.com/share/cebf106ada4f4ac789399a6eb27dd385)
+
+## Live Demo
+
+[Live Demo Link]
+
+## Built With
+
+- HTML
+- CSS
+- JS
+- WEBPACK
+- API service
+
+## Getting Started
+
+**To clone this project**
+**Run command: $git clone the reposiratory**
+**or press the green Code button in the upper right corner and choose to download from the Download ZIP link.**
+**To setup the webpack for this project**
+
+#### Install Webpack
+
+- Follow the instructions from the [getting started](https://webpack.js.org/guides/getting-started/#basic-setup) guide to set up the basics. Implement all the steps from _Basic Setup_ to _NPM Scripts_.
+- The next step is to add html file :
+  Follow the instructions from the [setting up HtmlWebpackPlugin](https://webpack.js.org/guides/output-management/#setting-up-htmlwebpackplugin) guide
+  Then
+
+```
+npm run build
+```
+
```

Comment:
Please add `npm install` and `npm test` commands.

Reply:
Thank you